### ISSUE TITLE
Does does not show an error after re introducing a syntax error

### ISSUE THREAD

**Issue by aoshi321** [gaearon/react-hot-boilerplate#99]
Hi

I've found when trying the project in the 'next' branch which uses react-hot-loader 3 the following to happen:

When I add a syntax error to a component it correctly show's me an error in the console.
When I fix the error it rebuilds correctly.
Adding the syntax error back in I get no errors in the console and renders without issue

This is happening consistently.

thanks

Aoshi

**Comment by calesce** [comment]
Now that we're using `webpack-hot-middleware` this is no longer an issue.

**Comment by vsheyanov** [comment]
@calesce I just checked out **next** branch, started the app and added an error in Counter.jsx
```
<h2>Counter: {{this.state.counter}</h2>
```
After seen an error screen and fixing this error I got this:
```
[HMR] Error: Cannot find module "./Counter"
```

**Comment by calesce** [comment]
@vsheyanov you're right, we needed Webpack's `NoErrorsPlugin`. That's now fixed in a26a65315dd36b48670b04ccb1a91643d185d52d.
